app/helpers/application_helper.rb

```diff
@@ -14,4 +14,8 @@ def cart_size
   def title(page_title)
     content_for :title, page_title.to_s
   end
+
+  def flash_class type
+    type == "notice" ? "alert alert-success" : "alert alert-danger"

```

Comment:
actually I don't see this valid assumption.
normally the class is `"alert alert-#{type}"`, but sometimes I get the type as alert so it becomes alert-alert class

Reply:
it's bootstrap classes so you can't use ` "alert alert-#{type}" ` it will end up with "alert alert-notice" & "alert alert-alert"

Reply:
I think `notice` and `alert` are the only odd cases. So the way to go is to use if for those 2 and otherwise set it to what is `alert alert-#{type}`

Reply:
notice -> info, alert -> danger otherwise the same